packages/react-components/src/components/VideoGallery/RemoteContentShareReactionOverlay.tsx

```diff
@@ -0,0 +1,244 @@
+        // Limit the number of active reactions of a certain type to 10
+        if (activeCount >= 10) {
+          latestReceivedReaction.current[userId] = {
+            id: combinedKey,
+            status: 'ignored'
+          };
+          return;
+        }
+
+        activeTypeCount.current[reaction.reactionType] += 1;
+        latestReceivedReaction.current[userId] = {
+          id: combinedKey,
+          status: 'animating'
+        };
+        const reactionMovementIndex = findFirstEmptyPosition();
+        visibleReactionPosition.current[reactionMovementIndex] = true;
+        setVisibleReactions([
+          ...visibleReactions,
+          {
+            reaction: reaction,
+            id: combinedKey,
+            reactionMovementIndex: reactionMovementIndex,
+            styleBucket: getReactionStyleBucket(reactionMovementIndex)
+          }
+        ]);
+        return;
+      },
+      [activeTypeCount, visibleReactions]
+    );
+
+    const removeVisibleReaction = (reactionType: string, id: string, index: number): void => {
+      setVisibleReactions(visibleReactions.filter((reaction) => reaction.id !== id));
+      visibleReactionPosition.current[index] = false;
+      activeTypeCount.current[reactionType] -= 1;
+      Object.entries(latestReceivedReaction.current).forEach(([userId, reaction]) => {
+        if (reaction.id === id) {
+          latestReceivedReaction.current[userId].status = 'completedAnimating';
+        }
+      });
+    };
+
+    // Update visible reactions when local participant sends a reaction
+    useEffect(() => {
+      if (localParticipant?.reaction) {
+        updateVisibleReactions(localParticipant.reaction, localParticipant.userId);
+      }
+    }, [localParticipant, updateVisibleReactions]);
+
+    // Update visible reactions when remote participants send a reaction
+    useEffect(() => {
+      remoteParticipants?.map((participant) => {
+        if (participant?.reaction) {
+          updateVisibleReactions(participant.reaction, participant.userId);
+        }
+      });
+    }, [remoteParticipantReactions, remoteParticipants, updateVisibleReactions]);
+
+    // Note: canRenderReaction shouldn't be needed as we remove the animation on the onAnimationEnd event
+    const canRenderReaction = (reaction: Reaction, id: string, movementIndex: number): boolean => {
+      // compare current time to reaction.received at and see if more than 4 seconds has elapsed
+      const canRender = Date.now() - getReceivedUnixTime(reaction.receivedOn) < REACTION_SCREEN_SHARE_ANIMATION_TIME_MS;
+
+      // Clean up the reaction if it's not in the visible reaction list
+      if (!canRender) {
+        removeVisibleReaction(reaction?.reactionType, id, movementIndex);
+      }
+
+      return canRender;
+    };
+
+    const containerHeight = hostDivHeight ?? 0;
+    const containerWidth = hostDivWidth ?? REACTION_START_DISPLAY_SIZE - REACTION_START_DISPLAY_SIZE;
```

Comment:
This has REACTION_START_DISPLAY_SIZE - REACTION_START_DISPLAY_SIZE which is 0?

Reply:
Yes. Thank you.